.ci/monolithic-linux.sh

```diff
@@ -33,6 +33,8 @@ function at-exit {
 
   mkdir -p artifacts
   ccache --print-stats > artifacts/ccache_stats.txt
+  cp "${BUILD_DIR}"/.ninja_log artifacts/.ninja_log
+  ls artifacts/
```

Comment:
Should include hidden files, since we are uploading these too according to the other PR.
Might look prettier in list form.
Might be worth doing it recursively.
```suggestion
  ls -laR artifacts/
```

Why not do the same on the windows job though?

Reply:
Leftover from testing. I've removed it. The github actions workflow should print out the file list.